lib/ast-converter.js

```diff
@@ -776,6 +786,13 @@ module.exports = function(ast, extra) {
                     }
                 }
 
+                /**
+                 * Prefix FunctionDeclarations within TypeScript namespaces with "TS"
+                 */
+                if (node.parent && node.parent.kind === SyntaxKind.ModuleBlock) {
+                    functionDeclarationType = "TS" + functionDeclarationType;
```

Comment:
TSNamespaceFunctionDeclaration?

Is there any other location where this name might be used?